.github/workflows/docs-verify.yml

```diff
@@ -5,12 +5,15 @@ on:
 
 jobs:
   verify_docs_build:
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.11']
```

Comment:
Similar to a previous comment, should we add here the rest of the python versions?

Reply:
For docs, I don't think we need 3.10, as I figure we'll update straight to 3.11 when we're ready.